apps/odyssey-stats/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+	"extends": "@automattic/calypso-typescript-config/mixed-package.json",
+	"compilerOptions": {
+		"rootDir": "../..",
+		"noEmit": true
```

Comment:
We use babel to transpile TypeScript, so we set `noEmit` as `true` here.

https://www.typescriptlang.org/tsconfig#noEmit